fault/tester.py

```diff
@@ -1,4 +1,4 @@
-import magma
+iclockmport magma
```

Comment:
oops :D

Reply:
heh, that's what I get for no tests. Fixed, and pushed a test for the string methods.